### PR TITLE
Part

### PR DESCRIPTION
**Pull request by TehanovAnton**
#9 I create Par and CarPart model, add associations. Car can have many parts

### CODE REVIEW

**Comment by TehanovAnton** [comment]
#9 Add business models
I create Par and CarPart model. Car can have many parts

**Comment by dmokreckiy** [comment]
@TehanovAnton Now I've noticed that you add you description as a comment. Please use description section of PR itself. Ask me or Yuri how to do that.
Still no reviewer and assignee

File: app/controllers/omniauth_callbacks_controller.rb [hunk 1]
```diff
@@ -1,14 +1,15 @@
...
+    if @user.persisted?
+      flash[:notice] = I18n.t 'devise.omniauth_callbacks.success', kind: 'Google'
+      @user.confirm unless @user.confirmed?
+      sign_in_and_redirect @user, event: :authentication
+    else
+      session['devise.google_data'] = request.env['omniauth.auth'].except('extra') # Removing extra as it can overflow some session stores
+      redirect_to new_user_registration_url, alert: @user.errors.full_messages.join("\n")
+    end
   end
 end
```
Comment (dmokreckiy) [inline app/controllers/omniauth_callbacks_controller.rb]: End line missing

File: db/migrate/20211229173008_create_car_parts.rb [hunk 2]
```diff
@@ -0,0 +1,14 @@
+class CreateCarParts < ActiveRecord::Migration[6.1]
+  def change
+    create_table :car_parts, id: false do |t|
```
Comment (dmokreckiy) [inline db/migrate/20211229173008_create_car_parts.rb]: `cars_parts` no?

**Review by dmokreckiy — CHANGES_REQUESTED** [review]
Please recheck `cars_parts` table name

**Review by yury-yaroshyk-itechart — APPROVED** [review]

**Pull request merged**